### PR TITLE
Remove support for SLES11 and update tests to just on openSUSE

### PR DESCRIPTION
**Pull request by dheerajd-msys**
Signed-off-by: dheerajd-msys <dheeraj.dubey@msystechnologies.com>

## Description
- Remove support for SLES11 and update for Opensuse-Leap15
## Related Issue
Fixes https://github.com/chef/chef/issues/9180

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [ ] I have run the pre-merge tests locally and they pass.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] All commits have been signed-off for [the Developer Certificate of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

**Comment by tas50** [comment]
You probably want to delete the Insserv provider and provider spec as well.

**Comment by dheerajd-msys** [comment]
Yes, Let me reset the commit and keep the PR on hold/close until we decide the platforms where `insserv` is supported.

File: spec/functional/resource/insserv_spec.rb [hunk 1]
```diff
@@ -1,206 +0,0 @@
...
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-
-require "spec_helper"
-require "functional/resource/base"
-require "chef/mixin/shell_out"
-require "fileutils"
-
-describe Chef::Resource::Service, :requires_root, :sles11 do
```
Comment (tas50) [inline spec/functional/resource/insserv_spec.rb]: Can you see if this can potentially run on Ubuntu?

Comment (dheerajd-msys) [reply spec/functional/resource/insserv_spec.rb]: Sure, @tas50 will test that.

Comment (dheerajd-msys) [reply spec/functional/resource/insserv_spec.rb]: Hi @tas50 We've checked tests are not running in Ubuntu16 & Ubuntu18 but with little change in `initest` script that are supporting for the tests they run on Opensuse15.

**Comment by tas50** [comment]
@dheerajd-msys can you pull out just the changes to the sysctl resource and submit a PR for that so we can at least get that part shipped in Chef 16?

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -169,7 +169,7 @@ module Shell
   config.filter_run_excluding linux_only: true unless linux?
   config.filter_run_excluding aix_only: true unless aix?
   config.filter_run_excluding suse_only: true unless suse?
-  config.filter_run_excluding sles11: true unless sles11?
+  config.filter_run_excluding opensuse15: true unless opensuse15?
```
Comment (tas50) [inline spec/spec_helper.rb:172]: can we just change this to opensuse so we catch 16 as well where this most likely continues to work. If not when we upgrade those testers in buildkite these tests will just disappear.

Comment (dheerajd-msys) [reply spec/spec_helper.rb]: Updated @tas50.

File: .expeditor/verify.pipeline.yml [hunk 3]
```diff
@@ -101,6 +103,8 @@ steps:
   commands:
     - /workdir/scripts/bk_tests/bk_container_prep.sh
     - zypper install -y cronie
+    - zypper install -y cron
```
Comment (tas50) [inline .expeditor/verify.pipeline.yml:106]: You shouldn't need cron if you have cronie, which is the default in opensuse. Also you can stick these all in a single line which runs faster.

File: spec/support/platform_helpers.rb [hunk 4]
```diff
@@ -231,8 +231,7 @@ def selinux_enabled?
 end
 
 def suse?
-  ::File.exists?("/etc/SuSE-release") ||
```
Comment (tas50) [inline spec/support/platform_helpers.rb:234]: Will this work on SLES 12 still? /etc/os-release is fairly new

File: .expeditor/verify.pipeline.yml (hunk 3 above)
Comment (dheerajd-msys) [reply .expeditor/verify.pipeline.yml:106]: Thanks @tas50 , Going to update that.

**Pull request merged**